core/napi.rs

```diff
@@ -0,0 +1,502 @@
+#![allow(non_camel_case_types)]
```

Comment:
Missing copyright

Reply:
Also, do we have to use non camel case types? I guess it's expected by NAPI specified in Node, if so a comment here would be helpful